### PR TITLE
create storage account module

### LINKED ISSUE #30: Create Storage Account Helpers
Create Azure Storage Account helper functions that returns Storage Account resource and check if the resource exists

Examples can be found in https://github.com/Azure/benchpress/tree/main/BenchPress/Helpers/Azure

### PR DESCRIPTION
**Pull request by polatengin**
closes #30

### CODE REVIEW

File: Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      System.String
+
+    .OUTPUTS
+      ConfirmResult
+  #>
+  [CmdletBinding()]
+  [OutputType([ConfirmResult])]
+  param (
+    [Parameter(Mandatory=$true)]
+    [string]$ServerName,
```
Comment (marclerwick) [inline Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1:35]: This parameter should be renamed to `Name`.

**Review by marclerwick — CHANGES_REQUESTED** [review]

File: Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1 (hunk 1 above)
Comment (polatengin) [reply Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1:35]: good catch, thanks 👍

File: Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1 [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    [string]$Name,
+
+    [Parameter(Mandatory=$true)]
+    [string]$ResourceGroupName
+  )
+  Begin {
+    $ConnectResults = Connect-Account
+  }
+  Process {
+      $Resource = Get-AzStorageAccount -ResourceGroupName $ResourceGroupName -Name $ServerName
```
Comment (msha1026) [inline Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1:44]: $ServerName should be $Name

File: examples/StorageAccount/StorageAccount.Tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+Describe 'Verify Storage Account Exists' {
+    it 'Should contain a Storage Account with the given name' {
+        #arrange
+        $rgName = 'rg-test'
+        $serverName = 'azbenchpressstorage'
+
+        #act
+        $result = Confirm-AzBPStorageAccount -ResourceGroupName $rgName -Name $serverName
```
Comment (msha1026) [inline examples/StorageAccount/StorageAccount.Tests.ps1:12]: Change $serverName names in this file to $name or $stgAcctName

File: examples/StorageAccount/StorageAccount_Example.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# How To Run StorageAccount.Tests.ps1
+
+`StorageAccount.Tests.ps1` contains examples of using the `Confirm-AzBPStorageAccount` cmdlet.
+
+## Pre-Requisites
```
Comment (msha1026) [inline examples/StorageAccount/StorageAccount_Example.md:5]: Update this doc to reuse the setup instructions at the root of the examples folder. Please reference main branch for how other docs are doing this.

**Review by msha1026 — CHANGES_REQUESTED** [review]

File: Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1 (hunk 2 above)
Comment (polatengin) [reply Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1]: good catch! thanks 👍

File: examples/StorageAccount/StorageAccount_Example.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+   ```Powershell
+    New-AzResourceGroupDeployment -ResourceGroupName "<your-resource-group-name>"`
+    -TemplateFile ".\storageAccount.bicep"
+   ```
+
+1. Update `StorageAccount.Tests.ps1` variables to point to your expected resources:
+
+   - `rg-test` -> `your-resource-group-name`
+   - `sampleStorageAccount` -> `your-storage-account-name`
```
[the remark this answers is not in the document]
Comment (polatengin) [reply examples/StorageAccount/StorageAccount_Example.md]: good catch! thanks 👍

File: examples/StorageAccount/StorageAccount.Tests.ps1 (hunk 3 above)
Comment (polatengin) [reply examples/StorageAccount/StorageAccount.Tests.ps1]: good catch! thanks 👍

File: examples/StorageAccount/StorageAccount_Example.md (hunk 4 above)
Comment (polatengin) [reply examples/StorageAccount/StorageAccount_Example.md:5]: I don't see a difference between the Pre-Requisites and Common Steps sections in this file and Readme in the root of the examples folder. can you be more specific, please?

**Review by msha1026 — APPROVED** [review]

**Pull request merged**